app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -56,6 +56,7 @@
     private final static String ENFORCE_SECURE_ENDPOINT = "cc-enforce-secure-endpoint";
     private final static String ALLOW_SPACE_IN_SELECT_CHOICES = "cc-allow-space-in-select-choices";
     private final static String USE_EXPRESSION_CACHING_IN_FORMS = "use-expression-caching-in-forms";
+    private final static String ALLOW_MANUAL_FORM_QUARANTINE = "cc-allow-manual-form-quarantine";
```

Comment:
I think this needs to be behind the user visible / controllable developer preferences, not the "Special Dimagi" level preferences.

Manual quarantining is a support tool that we need non-Dimagi personnel to be able to trigger to recover a clean state on their devices.